x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/empty_state.test.tsx

```diff
@@ -21,12 +21,24 @@ describe('EmptyState', () => {
     expect(wrapper.find(EuiEmptyPrompt)).toHaveLength(1);
   });
 
-  it('sends telemetry on create first engine click', () => {
-    const wrapper = shallow(<EmptyState />);
-    const prompt = wrapper.find(EuiEmptyPrompt).dive();
-    const button = prompt.find(EuiButton);
+  describe('CTA Button', () => {
+    let wrapper: ShallowWrapper;
+    let prompt: ShallowWrapper;
+    let button: ShallowWrapper;
+
+    beforeEach(() => {
+      wrapper = shallow(<EmptyState />);
+      prompt = wrapper.find(EuiEmptyPrompt).dive();
+      button = prompt.find('[data-test-subj="EmptyStateCreateFirstEngineCta"]');
+    });
+
+    it('sends telemetry on create first engine click', () => {
+      button.simulate('click');
+      expect(mockTelemetryActions.sendAppSearchTelemetry).toHaveBeenCalled();
+    });
 
-    button.simulate('click');
-    expect(mockTelemetryActions.sendAppSearchTelemetry).toHaveBeenCalled();
+    it('sends a user to engine creation', () => {
+      expect(button.prop('to')).toEqual('/engine_creation');
+    });
```

Comment:
It's not clear to me why this had to be split up into separate `it` blocks / beforeEach's etc. Just IMO, neither the component nor the test descriptions are distinct enough to warrant this extra amount of cruft. I would probably have opted to added a single line of `expect(button.prop('to')).toEqual('/engine_creation');` to the original test and called it a day.